### PR TITLE
docs: add outerbounds to showcase

### PR DESCRIPTION
**Pull request by emattia**

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1352,6 +1352,14 @@ const Users: User[] = [
     source: 'https://github.com/pingcap/ossinsight',
     tags: ['opensource', 'design'],
   },
+  {
+    title: 'Outerbounds',
+    description: 'A documentation site authored from jupyter notebooks using nbdoc and docusaurus.',
+    preview: require('./showcase/outerbounds.png'),
+    website: 'https://outerbounds.com',
+    source: 'https://github.com/outerbounds/nbdoc',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1360]: Please only add the repo with Docusaurus-related code.

Comment (emattia) [reply website/src/data/users.tsx]: Thank you - I've removed the non-Docusaurus repo and set `source: null`.

**Review by Josh-Cena — APPROVED** [review]
Thanks!

I'm holding off because I think it deserves a place in the favorites, but our favorites is already getting crowded. Do you have plans to share your success about integrating Docusaurus with Jupyter/nbdoc?

**Comment by Josh-Cena** [comment]
I'd personally love that! @slorber can give a more authoritative answer here, but in any case we can publicize your published post at least.

**Comment by emattia** [comment]
Sounds great @Josh-Cena - will follow-up here when we are ready to publish the post. Thanks again!

**Comment by slorber** [comment]
Great site 👍 interesting layout

> Would you be interested in posting something like this on Docusaurus blog?

We plan to do that yes, so if you are using Docusaurus in a fancy and inspiring way and have something to say about it, that's definitively possible. We'd like to publish case studies as blog posts, but not case studies that are just describing Docusaurus features and paraphrasing our doc.

A good example that we almost published on our blog: https://www.courier.com/blog/how-we-built-our-documentation/

**Pull request merged**